DQM/Integration/python/clients/beampixel_dqm_sourceclient-live_cfg.py

```diff
@@ -30,14 +33,17 @@
 process.load("Configuration.StandardSequences.GeometryRecoDB_cff")
 process.load('Configuration.StandardSequences.MagneticField_AutoFromDBCurrent_cff')
 process.load("Configuration.StandardSequences.RawToDigi_Data_cff")
-process.load("DQM.Integration.config.FrontierCondition_GT_cfi")
```

Comment:
this is causing a crash on the playback where we are testing this PR.
please revert this line back and propagate the fix to 75 and 76 as well.